### DISCUSSION TITLE
[Question]: Do we support cs1.5 on Linux?

### DISCUSSION THREAD

**Discussion by rickyma** [theAsmodai/metamod-r#77] [Q&A] [answered]
### Tell us more about your Question

Upon visiting the link https://github.com/theAsmodai/metamod-r/wiki/Supported-games, I noticed that it states "There are no executable files for Linux" for Counter-Strike 1.5. 

However, I found the `metamod_i386.so` file in the current release binary package. So I'm wondering if the documentation hasn't been updated, or if there's something I might have missed? Do we support cs1.5 on Linux?

**Comment by stamepicmorg** [comment, marked as answer]
I think we need update this list and retest the most games :)
